scripts/gui/thrive_gui.html

```diff
@@ -97,6 +97,14 @@
         </div>
 	  </div>
 	  <div id="pauseButton"><div id="pauseIcon"></div></div>
+	  
+      <!-- Population bar -->
+      <div class="population">
```

Comment:
If I saw this correctly there are only 6 spaces here. There should be 8 to match up with the surrounding tags.